tests/unit/cn/move_test.c

```diff
@@ -0,0 +1,330 @@
+/* SPDX-License-Identifier: Apache-2.0 */
```

Comment:
Naming this file cn_move_test makes the test identifier more unique. Just a thought

Reply:
Okay, it's already under cn/ directory, so thought cn_ file name prefix would be repetitive.

Reply:
Normally I would agree with you, but since Meson test names exist globally, I don't think there is harm in changing it.

Remember that the generated executable is `builddir/tests/unit/<my test>`

Reply:
Good point, will change it.